packages/liveblocks-node/test-d/client.test-d.ts

```diff
@@ -0,0 +1,15 @@
+import { expectType } from "tsd";
+import { Liveblocks } from "../src/client";
+import type { PlainLsonObject, JsonObject } from "@liveblocks/core";
+
+async () => {
+  const client = new Liveblocks({ secret: "sk_xxx" });
+
+  expectType<PlainLsonObject>(await client.getStorageDocument("document-id"));
+  expectType<PlainLsonObject>(
+    await client.getStorageDocument("document-id", "plain-lson")
+  );
+  expectType<JsonObject>(
+    await client.getStorageDocument("document-id", "json")
```

Comment:
This is perfect! Think of these as unit tests, but for our type definitions. What’s expressed here is that if you call `.getStorageDocument()` it will return a value of type `PlainLsonObject` (exactly the same as from `core`. If this function or return type would ever change, then this “unit test” can fail in the future, which then might indicate a breaking change.

The value of these tests grow over time, as we approach 100% type coverage with them. They are especially useful for people making changes in `@liveblocks/core`, where it may be unclear what the effect on `@liveblocks/node`’s types are.